src/components/figures/figure_chart/figure_chart.xml

```diff
@@ -6,7 +6,7 @@
       t-on-contextmenu.prevent.stop="(ev) => !env.isDashboard() and this.onContextMenu(ev)">
```

Comment:
this is probably where you should add a condition on `env.model.getters.isReadonly()`. You prevent the possibility to open the menu when on readonly mode.